web_viewer/client/octree_viewer.ts

```diff
@@ -308,6 +309,27 @@ export class OctreeViewer {
       });
   }
 
+  public setMoving(value: boolean) {
+    if (value) {
+      for (const nodeId of Object.keys(this.loadedData)) {
+        if (
```

Comment:
Seems like confusing logic. Any reason you can't invert the condition here and put the last line inside the `if`?

Reply:
I cannot wrap my head around the inverted condition.... The current code reads clearer to me. I added a bit of comments to make it better understandable. wdyt?

Reply:
I just meant this (DeMorgan's law ftw):

```
  for (const nodeId of Object.keys(this.loadedData)) {
    if (nodeId.length > maxLevelToDisplay &&
        this.loadedData[nodeId].threePoints) {
      this.loadedData[nodeId].threePoints.visible = false;
    }
  }
```
As a reader, if I want to understand when the line below *does* get executed, I already have to mentally invert the condition. To me, it seems weird to say, "If such-and-such is true, do nothing. Otherwise, do this thing."

Reply:
okay, done now. I guess my brain finally settled in.